src/Facade/Services/PurchaseOrderFacadeService.cs

```diff
@@ -97,6 +97,27 @@ public class PurchaseOrderFacadeService :
             }
         }
 
+        public async Task<IResult<ProcessResultResource>> EmailSupplierAss(int orderNumber)
```

Comment:
Does this need to be async (and return a task for that matter?) Don't see any await keyword in here